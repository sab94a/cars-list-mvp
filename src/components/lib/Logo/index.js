// @flow

import React from 'react';
import logo from './logo.png';
import styles from './index.module.scss';

const Logo = () => (
    <img className={ styles.root } alt="Logo" src={ logo } />
);

export default Logo;
