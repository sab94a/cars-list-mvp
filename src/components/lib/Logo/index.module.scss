@import 'components/lib/variables.scss';

.root {
    display: block;
    height: $panel-height - $offset-lg * 2;
    align-items: center;
}