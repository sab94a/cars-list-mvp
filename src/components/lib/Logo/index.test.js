import React from 'react';
import { mount } from 'enzyme';
import toJson from 'enzyme-to-json';
import Logo from './'

describe('<Logo />', () => {
    it('Should render correctly', () => {
        const component = mount(<Logo />)

        expect(toJson(component)).toMatchSnapshot();
    });
});
