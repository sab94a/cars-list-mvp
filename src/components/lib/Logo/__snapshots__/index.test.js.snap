// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Logo /> Should render correctly 1`] = `
<Logo>
  <img
    alt="Logo"
    className="root"
    src="logo.png"
  />
</Logo>
`;
